bibtex-actions.el

```diff
@@ -274,14 +300,29 @@ offering the selection candidates."
          ((string= extension (or "org" "md")) "Notes")
           (t "Library Files")))))
 
+(defun bibtex-actions-latex--local-bib-files ()
+  "Local bibliographic for latex retrieved using reftex"
+  (reftex-access-scan-info t)
```

Comment:
This causes a rescan which is fast enough for me and has the advantage that if a bib file has been added or removed, that is detected.

Reply:
Check the docstring.

Reply:
> Check the docstring.

Thanks, fixed